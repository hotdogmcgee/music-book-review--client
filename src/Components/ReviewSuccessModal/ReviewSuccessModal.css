.review-success-modal {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  padding: 20px;
  height: auto
}

.review-success-modal a {
  padding: 25px;
  font-size: 25px;
  color: rgba(82, 96, 210, 1)
}
