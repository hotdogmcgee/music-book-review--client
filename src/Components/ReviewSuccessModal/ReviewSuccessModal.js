import React from "react";
import "./ReviewSuccessModal.css";
import { Link } from "react-router-dom";
import { Button } from '../Utils/Utils'

export default class ReviewSuccessModal extends React.Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    const { show, handleClose } = this.props;
    const showHideClassName = show
      ? "modal display-block"
      : "modal display-none";
    return (
      <div className={showHideClassName}>
        <section className="modal-main review-success-modal">
          <Link to="/">Go to Home Page</Link>
          <Button onClick={handleClose}>Close</Button>
        </section>
      </div>
    );
  }
}
