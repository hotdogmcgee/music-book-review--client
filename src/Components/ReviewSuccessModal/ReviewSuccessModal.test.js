import React from "react";
import ReviewSuccessModal from "./ReviewSuccessModal";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("ReviewSuccessModal component", () => {
  it("renders a ReviewSuccessModal by default", () => {
    const wrapper = shallow(<ReviewSuccessModal />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
