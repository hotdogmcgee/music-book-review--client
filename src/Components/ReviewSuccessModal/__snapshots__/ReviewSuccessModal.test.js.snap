// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewSuccessModal component renders a ReviewSuccessModal by default 1`] = `
<div
  className="modal display-none"
>
  <section
    className="modal-main review-success-modal"
  >
    <Link
      to="/"
    >
      Back to Home Page
    </Link>
    <button>
      Close
    </button>
  </section>
</div>
`;
